tests/specs/check/unstable_suggestion/main.out

```diff
@@ -0,0 +1,10 @@
+Check file:///[WILDLINE]main.ts
+error: TS2551 [ERROR]: Property 'dlopen' does not exist on type 'typeof Deno'. Did you mean 'open'? 'Deno.dlopen' is an unstable API. Did you mean 'open'? If not, try changing the 'lib' compiler option to include 'deno.unstable' or add a triple-slash directive to the top of your entrypoint (main file): /// <reference lib="deno.unstable" />
```

Comment:
I'll assume this was generated and you C&P it here. TS2551 does appear to be the correct error code, but I'm not sure where these are defined. 👍